src/components/SVGImage.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class SVGImage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loaded: false,
+      error: false,
+    };
+
+    this.image = new Image();
+    this.image.onload = () => {
+      if (this.props.onLoad) this.props.onLoad(this.image);
+      this.setState({
+        loaded: true,
+      });
+    };
+    this.image.onerror = (err) => {
+      if (this.props.onError) this.props.onError(err);
+      this.setState({
+        error: true,
+      });
+    };
+
+    if (this.props.src) {
+      this.image.src = this.props.src;
+    } else {
+      this.state.loaded = false;
+      this.state.error = true;
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.src !== this.image.src) {
+      this.image.src = nextProps.src;
+    }
+  }
+
+  render() {
+    if (this.state.loaded) {
+      return (
+        <image className="svg-image"
+          xlinkHref={this.image.src}
+          width={this.image.width}
+          height={this.image.height}
+          x={(this.image.width * -0.5)+'px'}
+          y={(this.image.height * -0.5)+'px'} />
+      );
+
+    //TODO: review loading and error indicators.
```

Comment:
Can we check with @beckyrother on what's our Go-To symbol/sigil/icon/glyph/emoji/sacred mark for indicating "Something went terribly wrong" ? I think we already settled on a proper loading symbol too, didn't we?

None of the symbols I can think of works:
⚠️ - generic warning that makes errors seems more exciting that they should be; ❌  generic error/cancel symbol that can be confused for mathematical multiplication; ☠️  - incredibly confusing indicator that our website is either poisonous or stolen by pirates.